src/InboundServer/handlers.js

```diff
@@ -191,12 +221,18 @@ const postTransfers = async (ctx) => {
     // so it is safe to return 202
     ctx.response.status = 202;
     ctx.response.body = '';
+    histTimerEnd({ success: true });
 };
 
 /**
  * Handles a PUT /participants/{ID}. This is a response to a POST /participants request
  */
 const putParticipantsById = async (ctx) => {
+    const histTimerEnd = Metrics.getHistogram(
+        'inbound_put_participants_id',
```

Comment:
Any reason that you are not using the information that you set in the Route Map instead....it seems that this is using the same metric names for the event framework?

Reply:
decided not to go with this option as name in route map requires the path to access